webapp/src/selectors/index.tsx

```diff
@@ -16,6 +16,10 @@ export const getLinkModalState = (state: any): LinkProjectModalState => {
     return state[pluginPrefix].openLinkModalSlice;
 };
 
+export const getCreateTaskModalState = (state: any): CreateTaskModalState => {
```

Comment:
Change the type of `state` from any to the correct type.

Reply:
already fixed